src/components/portals/README.md

```diff
@@ -0,0 +1,100 @@
+### Portals and Finsemble
+
+Portals in finsemble work just like they do in the browser. `window.open`, and voila. There are a couple of things that we can do in finsemble that are slightly different from how things work in a browser.
+
+### Definitions
+Child Window - The object returned by `window.open`.
+
+FinsembleWindow - A low level API to the physical window that your finsemble component lives inside of. This API is currently private, but we're exposing it in this POC and may expose it properly in a future release.
+
+### Operating on child windows
+Child windows are is comparatively limited to the `finsembleWindow`. Consider the following code:
+
+```javascript
+const cw = window.open('about:blank');
+cw.focus();
+```
+
+This does nothing. The focus method doesn't work in the browser. To get around this limitation, you will need to work with the `finsembleWindow`. How do you get the finsembleWindow?
+
+
+```javascript
+const childWindow = window.open(undefined, undefined, `width=${width},height=${height},top=${top},left=${left}`);
+
+/**
+ * finsembleWindow doesn't exist until FSBL is online. If the hide/show button
+ * was clicked before that happened (small chance, but possible), these functions
+ * would fail. We don't want that.
+ *
+ * Here, we return the finsembleWindow if it exists; otherwise, we wait.
+ */
+const getFinsembleWindowFromChild = () => {
+  if (childWindow.finsembleWindow) return Promise.resolve(childWindow.finsembleWindow);
+  return new Promise((resolve) => {
+      childWindow.addEventListener("FSBLReady", () => {
+          // because this code is gated behind the FSBLReady listener,
+          // it won't run when your app is running in a browser without
+          // finsemble.
+          resolve(childWindow.finsembleWindow);
+      });
+  })
+}
+
+const childFinsembleWindow = await getFinsembleWindowFromChild();
+
+```
+
+### Focusing a child window
+
+Focusing a child window is simple, provided you've followed the steps above.
+
+```javascript
+childFinsembleWindow.focus();
+```
+
+### Retrieving the bounds of a child window
+
+Retrieving the bounds of a child window can be done in a couple of ways.
+
+In the browser, you can access `childWindow.screenX` and `childWindow.screenY` (NOTE: These are childWindows, not childFinsembleWindows).
+
+Because of a bug in Electron, these values are off due to scaling issues. Instead, you should use the `finsembleWindow`.
```

Comment:
i) I assume this won't be merged, so this isn't a big deal, but...
a) when we say a bug exists, we should link to it
b) I don't think window.screenX and window.screenY have a bug in chromium or electron.  The windows operating system handles scaling with a scaling factor, and that can lead to unexpected values.  Please let me know if I'm wrong here.

Reply:
Changed wording.